src/button/button.py

```diff
@@ -71,35 +74,49 @@ def __setKwargs(self, **kwargs):
         :param 3-tuple bg_normal: color for normal background
         :param 3-tuple bg_hover: color for background when hover
         :param 3-tuple bg_pressed: color for background when pressed
+        :param 3-tuple bg: color for all the backgrouds
         :param int bd: border size
         :param 3-tuple bd_color: color for the border
         :param str text: text to display on button
         :param dict text_font: {file: font family, size: font size}
         :param 3-tuple text_color: color for the text
+        :param str state: button state [enabled, disabled]
         """
 
         # Validate kwargs
         attr = [
             "bg_normal", "bg_hover", "bg_pressed",
             "bd", "bd_color",
             "text", "text_font", "text_color",
-            "pos", "size"
+            "pos", "size", "state", "bg"
         ]
         for kwarg in kwargs:
             if kwarg not in attr:
                 raise ValueError(f"{kwarg} is not a valid attribute")
 
-        # If a parameter has changed
-        # Update rect
+        # Special parameters
         if "pos" in kwargs:
             self.rect.x, self.rect.y = kwargs.pop("pos")
         if "size" in kwargs:
             self.rect.w, self.rect.h = kwargs.pop("size")
+        if "state" in kwargs:
+            state = kwargs.pop("state")
+            if state == "disabled":
+                self.state = State.disabled
+            elif state == "enabled":
+                self.state = State.normal
+        if "bg" in kwargs:
+            bg = kwargs.pop("bg")
+            self.bg_normal = bg
+            self.bg_hover = bg
+            self.bg_pressed = bg
```

Comment:
I think you forgot to use the index of `bg` list?

Reply:
I don't understand the question... Which index?

Reply:
`bg[0]`, `bg[1]`, ...
`bg` is the entire list, don't you need to choose the right one for each background?

Reply:
It's meant to apply the same background color to every backgrounds!
`Button(bg=(10, 10, 50))` > `bg_normal = bg_hover = bg_pressed`

Reply:
So, for now, the different backgrounds are always the same?

Reply:
So, for now, the different backgrounds are always the same?

Reply:
No, but you have a simple solution to do it :smile:
If you want a pressed background of color C1 and all the other backgrounds of color C2:
`Button(bg=C2, bg_pressed=C1)`
If you want all the backgrounds of color C1:
`Button(bg=C1) == Button(bg_normal=C1, bg_hover=C1, bg_pressed=C1)`

Reply:
Ah ok, I didn't think about that type of use!